tests/mysql_client_test.c

```diff
@@ -1538,8 +1537,7 @@ static void test_prepare()
       fprintf(stdout, "\n\t tiny   : %d (%lu)", tiny_data, length[0]);
       fprintf(stdout, "\n\t short  : %d (%lu)", small_data, length[3]);
       fprintf(stdout, "\n\t int    : %d (%lu)", int_data, length[2]);
```

Comment:
The size specifier for `char` is `hh`.
But since varargs smaller than `int` get auto-promoted to `int`, I won’t judge it here (or press for `h` support in `my_vsnprintf`).

This “`long`” type applied on an `int` data itches me, though –
They probably meant Maria-SQL `INT`, which is better defined with `int32_t` (and format with `"%"PRId32`) to strictly match `INT4`.